server/util/tracer.js

```diff
@@ -0,0 +1,83 @@
+const { registerInstrumentations } = require('@opentelemetry/instrumentation');
+const opentelemetry = require('@opentelemetry/api');
+
+// Not functionally required but gives some insight what happens behind the scenes
+// const { diag, DiagConsoleLogger, DiagLogLevel } = opentelemetry;
+// diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.INFO);
+
+const { AlwaysOnSampler, SamplingDecision, SimpleSpanProcessor } = require('@opentelemetry/sdk-trace-base');
+const { GraphQLInstrumentation } = require('@opentelemetry/instrumentation-graphql');
+const { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-proto');
+const { ConsoleSpanExporter, NodeTracerProvider } = require('@opentelemetry/sdk-trace-node');
+const { Resource } = require('@opentelemetry/resources');
+const { WinstonInstrumentation } = require('@opentelemetry/instrumentation-winston');
+const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+
+const Exporter = (process.env.EXPORTER || '').toLowerCase().startsWith('c') ? ConsoleSpanExporter : OTLPTraceExporter;
```

Comment:
I've included the ConsoleSpanExporter for local use. Starting the ui server locally with the "EXPORTER=console" env var will log spans for verification. We can remove after we refine this a bit more.

Reply:
Would it be possible to name this environment variable something more descriptive? Like `OPENTELEMETRY_EXPORTER_TYPE`?

Reply:
Sure, this is temporary but I can update